src/components/Error/NotFoundPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Row, Col, Button } from 'reactstrap';
+import { isExpectedPath } from 'kpmp-common-components';
+ 
+export function getBackToHomePath(pathname) {
```

Comment:
GetBackToHomePath was originally created to handle navigation between explorer and repository.  The function may still be useful here, but we don't want it to go to either /explorer or /repository, just /.

Could go two routes here.

1. Keep the getBackToHomePath function, just returning '/' in it. This would allow us to easily modify the logic if we deem important later on.
2. Remove the function all together, and thus just giving the '/' directly to the onClick button handler.